sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/DataFeedTests.cs

```diff
@@ -39,8 +39,8 @@ public void CreateDataFeedValidatesArguments()
                 IngestionSettings = ingestionSettings
             };
 
-            Assert.That(() => adminClient.CreateDataFeedAsync(null), Throws.InstanceOf<ArgumentNullException>());
-            Assert.That(() => adminClient.CreateDataFeed(null), Throws.InstanceOf<ArgumentNullException>());
+            Assert.That(() => adminClient.CreateDataFeedAsync(null, cancellationToken: default), Throws.InstanceOf<ArgumentNullException>());
```

Comment:
Why does this need to change?

Reply:
This call was ambiguous reference between protocol and convenience method

Protocol Method 
```C#
public virtual async Task<Response> CreateDataFeedAsync(RequestContent content, RequestContext context = null)
```

Convenience Method 
```C#
public virtual async Task<Response<DataFeed>> CreateDataFeedAsync(DataFeed dataFeed, CancellationToken cancellationToken = default)
```

To fix the test I passed CancellationToken.

Reply:
Yeah, we don't want to change the tests -- that means by adding the protocol methods we've broken client code.

Do you know why it is valid for a caller to pass null here?  It is to test that we're validating parameters?  I think this might be a mistake in the MA HLC client...

Reply:
This is just the unit test where we are validating parameter to check if the method throws `ArgumentNullException`, if we pass null parameter.

I don't think in this case we need to rename the method since protocol method takes `RequestContent` and convenience method take `DataFeed` as a parameter.

Reply:
Per @KrzysztofCwalina, it is fine to have an ambiguous call site when you pass `null` for some parameters, but lets disambiguate in the test by casting the null rather than passing the cancellation token, i.e. `adminClient.CreateDataFeedAsync((RequestContent)null)`

Reply:
Updated!